Vagrantfile

```diff
@@ -4,6 +4,7 @@
 build_boxes = [
   'appimage',
   'archlinux',
+  'centos-7',
```

Comment:
Is there a reason for adding centos-7 guest here?

Reply:
whoops! Left over from testing some stuff. Good catch ✅